File: B3/1267/cpp/1267.cpp
#include <iostream>
using namespace std;

int main(){
  int n, usingtime, i;
  int y = 0;
  int m = 0;
  cin >> n;

  for(i =0; i < n; i++){
    cin >> usingtime;
    y += ((usingtime / 30) + 1) * 10;
    m += ((usingtime / 60) + 1) * 15;
  }

  if(y > m) cout << "M " << m;
  else if(y < m) cout << "Y " << y;
  else cout << "Y M " << y;

  return 0;
}

File: B3/1267/python/1267.py
n = int(input())
usingtime = list(map(int,input().split()))
y = 0
m = 0

for i in usingtime:
  y += ((i // 30) + 1) * 10;
  m += ((i // 60) + 1) * 15;

if(y > m):
  print("M", m)
elif(y < m):
  print("Y", y)
else:
  print("Y M", y)